Products/Zuul/tree.py

```diff
@@ -236,7 +236,7 @@ def __init__(self, context):
         self.model_catalog = IModelCatalogTool(self.context)
 
     def getBrain(self, path):
-        return self.model_catalog.getBrain(path)
+        return self.model_catalog.global_catalog.getBrain(path)
```

Comment:
Should we maybe just put the fields in manually here, rather than use the legacy helper in platform code?